app/controllers/oauth/applications_controller.rb

```diff
@@ -29,7 +29,7 @@ def new
     end
 
     def create
-      @application = Doorkeeper::Application.new(application_params(@user))
+      @application = Doorkeeper::Application.new(app_params)
```

Comment:
Because only admins can create an oauth app now. This code should've changed when that behavior changed. Suggested by Dante ™️ .